src/docfx/build/dependency/DependencyResolver.cs

```diff
@@ -119,8 +121,11 @@ internal class DependencyResolver
 
             if (!string.IsNullOrEmpty(resolvedHref))
             {
-                var monikerQuery = !string.IsNullOrEmpty(moniker) ? $"view={moniker}" : "";
-                resolvedHref = UrlUtility.MergeUrl(resolvedHref, monikerQuery, fragment.Length == 0 ? "" : fragment.Substring(1));
+                if (!string.IsNullOrEmpty(moniker))
+                {
+                    queries["view"] = moniker;
+                }
+                resolvedHref = UrlUtility.MergeUrl(resolvedHref, "?" + string.Join('&', queries), fragment.Length == 0 ? "" : fragment.Substring(1));
```

Comment:
`queries` might be null here, can you make it such that `queries is always not null?

Reply:
sounds good